requirements-dev.txt

```diff
@@ -1,3 +1,6 @@
 httpretty
 mock
 pytest
+jsonschema
+pyyaml
+six
```

Comment:
These requirements are already listed in `setup.py`, no need to add them here.

Reply:
Apparently, the correct way is to put a line with a single `.` so that `pip install -r requirements-dev.txt` will automatically look in `setup.py` as well. I've pushed a new version reflecting this.